content/300-guides/050-database/100-developing-with-prisma-migrate/175-squashing-migrations.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Squashing migrations
+metaTitle: Squashing migrations
+metaDescription: How to squash multiple migration files into a single migration
+---
+
+<TopBlock>
+
+This guide describes how to squash multiple [migration files](/concepts/components/prisma-migrate#migration-history) into a single migration.
+
+</TopBlock>
+
+## About squashing migrations
+
+It is sometimes useful to squash either some or all migration files into a single migration. This guide will describe two scenarios where you may want to do this:
+
+- [Migrating cleanly from a development environment](#migrating-cleanly-from-a-development-environment) by squashing your local migrations into one before merging
+- [Creating a clean history in a production environment](#creating-a-clean-history-in-a-production-environment) by squashing all migrations into a single file
+
+In both cases, Prisma Migrate provides the tools for doing this, by using the [`migrate diff`](/reference/api-reference/command-reference#migrate-diff) command to compare two database schemas and output a single SQL file that takes you from one to the other. The rest of this guide gives detailed instructions on how to carry this out in these two scenarios.
```

Comment:
Hm, the first case does not even seem to be using `migrate diff` - or did I miss something?